docs/_layouts/blog.html

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Why is this a layout and not just `docs/blog.html`?

Reply:
Updated.